extensions/adobe/experience/offer-management/offer-placement.schema.json

```diff
@@ -57,10 +66,13 @@
   },
   "allOf": [
     {
-      "$ref": "#/definitions/offer-placement"
+      "$ref": "https://ns.adobe.com/xdm/data/record"
+    },
+    {
+      "$ref": "https://ns.adobe.com/experience/decisioning/placement"
     },
     {
-      "required": ["xdm:name", "xdm:channel", "xdm:componentType"]
+      "$ref": "#/definitions/offer-placement"
```

Comment:
@jbeckert , This schema is already deprecated and continues to be so. You are still making changes to the allOF section here. Is there a need for this or am I missing something.

Reply:
It was a mistake to mark the status as deprecated. What needed to be deprecated were the "xdm:" properties in this schema

Reply:
Let me amend the schema and bring back the status to "stable"

Reply:
Sounds good